### ISSUE TITLE
Extra padding at top of PdfViewer

### ISSUE THREAD

**Issue by JingYenLoh** [xcarpentier/rn-pdf-reader-js#24]
Hi, thank you for this library 😃

I have an issue with additional padding at the top of the component. Using the following code:

```js
const WebScreen = props => (
  <View style={styles.container}>
    <MyHeader name={props.navigation.getParam('name')} />
    <View style={{ flex: 1 }}>
      <PdfReader
        style={{ flex: 1, padding: 0, margin: 0 }}
        source={{ uri: props.navigation.getParam('uri') }}
      />
    </View>
    <BottomBar />
  </View>
);

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
  },
});

export default WebScreen;
```

I get the following result, with an extra 24px padding at the top.
![photo_2018-07-26_01-42-56](https://user-images.githubusercontent.com/22059033/43217876-6ad7ae9c-9075-11e8-93e3-96d94f2a94dc.jpg)

After some inspection, I found the extra padding to be due to [this line](https://github.com/xcarpentier/rn-pdf-reader-js/blob/4546288c624649758bac27d0a25a7b8c4041981b/index.js#L79). By changing `paddingTop` to 0, I was able to achieve the following:
![photo_2018-07-26_01-45-05](https://user-images.githubusercontent.com/22059033/43217933-8f265ba4-9075-11e8-8b11-b1a24eab905e.jpg)

Would it be possible for us to control paddingTop via a prop? I believe #22 is a related issue (have not tested on iOS)

**Comment by luongbeta2** [comment]
Hi @xcarpentier.  Please help me know when are you going to fix the bug ?

**Comment by vinitsood** [comment]
Any updates on this @xcarpentier? Awesome work btw :)

**Comment by xcarpentier** [comment]
Should be fixed now on v0.2.0
